src/esp/assets/GenericSemanticMeshData.cpp

```diff
@@ -153,9 +140,20 @@ GenericSemanticMeshData::buildSemanticMeshData(
         return (unsigned(color[0]) << 16) | (unsigned(color[1]) << 8) |
                unsigned(color[2]);
       };
+      constexpr const char Hex[]{"0123456789abcdef"};
+      auto colorAsStr = [](const Mn::Color3ub& color) -> std::string {
+        char out[] = "#______";
+        out[1] = Hex[(color.r() >> 4) & 0xf];
+        out[2] = Hex[(color.r() >> 0) & 0xf];
+        out[3] = Hex[(color.g() >> 4) & 0xf];
+        out[4] = Hex[(color.g() >> 0) & 0xf];
+        out[5] = Hex[(color.b() >> 4) & 0xf];
+        out[6] = Hex[(color.b() >> 0) & 0xf];
+        return std::string(out);
+      };
```

Comment:
I think I saw this exact piece of code somewhere else in this PR already :thinking: